src/main/java/eu/europeana/metis/sandbox/common/amqp/RecordMessageConverter.java

```diff
@@ -36,6 +40,7 @@ public Message toMessage(Object object, MessageProperties messageProperties) {
 
     Event recordEvent = (Event) object;
     Record record = recordEvent.getBody();
+    Optional<EventError> eventError = recordEvent.getEventError();
```

Comment:
I went away from using the Exception idea because at the end what I want to preserve is why the record event failed, so instead I created the EventError that will contain the error message and the stack trace of it, that I will be saving later. Now I removed the object mapper, and made a few changes to adjust the logic